index.html

```diff
@@ -4,9 +4,32 @@
     <meta charset="UTF-8">
     <meta name="Joe Wheelock" content="John Doe">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="/css/bootstrap/css/bootstrap.css">
-    <link rel="stylesheet" href="/css/styles.css"
+    <link rel="stylesheet" href="css/bootstrap/css/bootstrap.css">
+    <link rel="stylesheet" href="css/styles.css">
   </head>
   <body>
+    <div class="container-fluid yellow">
+      <div class="container">
+        <div class="row">
+          <header class="col-lg-12 header block"> [header] </header>
+        </div>
+        <div class="row">
+          <nav class="col-lg-12 nav block"> [nav] </nav>
+        </div>
+        <div class="row">
+          <content class="col-lg-9 content block"> [content] </content>
```

Comment:
Just wanted to point something out. The reason it is **red** in this section is because it is letting us know that `<content></content>` is not a real HTML tag. BUT, it will still work in your browser. Yet, it is not suggested to use it. You can call it `div` and use an `id="content` to make it unique. Just a suggestion. Other than that, great work!

Reply:
Ah, thanks for pointing that out! Changing it now.